pkg/storkctl/migration_test.go

```diff
@@ -424,6 +433,41 @@ func TestActivateDeactivateMigrations(t *testing.T) {
 	testCommon(t, cmdArgs, nil, expected, false)
 }
 
+func TestActivateDeactivateMigrationsAutoSuspend(t *testing.T) {
+	defer resetTest()
+	createMigratedDeployment(t)
+	createMigratedStatefulSet(t)
+	_, err := core.Instance().CreateNamespace(&v1.Namespace{ObjectMeta: metav1.ObjectMeta{Name: "adminNs"}})
+	require.NoError(t, err, "Error creating adminNs namespace")
+	createTestMigrationSchedule("test-migrationSchedule-1", "default-migration-policy",
+		"clusterPair1", []string{"dep"}, "adminNs", true, t)
+	createTestMigrationSchedule("test-migrationSchedule-2", "default-migration-policy",
+		"clusterPair1", []string{"sts"}, "adminNs", true, t)
+
+	//fake the dynamic client
+	scheme := runtime.NewScheme()
+	gvrToListKind := appregistration.GetSupportedGVR()
```

Comment:
the fakeDynamicClient has been reinitialised here because I want to also add the resources, so that the mock dynamic.List call actually returns the desired output.